src/main/java/br/ufs/hu/prevsep/web/api/dto/form/sepse/PageableDoctorFormDTO.java

```diff
@@ -68,12 +73,17 @@ public void setDtCriacaoEnd(LocalDate dtCriacaoEnd) {
     @Override
     public Predicate getQueryPredicate(QFormularioSepseMedicoEntity qEntity) {
         BooleanBuilder filter = new BooleanBuilder();
+        BooleanBuilder subfilter = new BooleanBuilder();
```

Comment:
Só é necessário criar esse objeto quando a lista de status for != null e tenha o tamanho > 0. Mover para dentro do if